### PR TITLE
[css-text-3] Tests for percentage text indent

### PR DESCRIPTION
**Pull request by frivoal**
Relates to the spec changes decided in
https://github.com/w3c/csswg-drafts/issues/2394

### CODE REVIEW

File: css/css-text/text-indent/text-indent-percentage-003.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  border-right: 10px solid white;
+  margin-right: 10px;
+  padding-right: 10px;
+}
+div {
+  box-sizing: border-box;
+  width: 120px;
+}
+.test div { text-indent: 50%; color: red; overflow: hidden; } /* overflow:hidden should not make any difference, but it does in some browsers */
+.ref div { text-indent: 50px; }
```
Comment (emilio) [inline css/css-text/text-indent/text-indent-percentage-003.html]: Could a test for `calc(px + %)` be added if it's not too much of a hassle? That's a case that gecko gets wrong.

File: css/css-text/text-indent/text-indent-percentage-003.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+section, div {
+  border-right: 10px solid white;
+  margin-right: 10px;
+  padding-right: 10px;
+}
+div {
+  box-sizing: border-box;
+  width: 120px;
+}
+.test div { text-indent: 50%; color: red; overflow: hidden; } /* overflow:hidden should not make any difference, but it does in some browsers */
```
[the remark this answers is not in the document]
Comment (emilio) [reply css/css-text/text-indent/text-indent-percentage-003.html]: I guess the browser is Gecko looking at the test locally. I'm not sure what's going on there. I'd file and cc @MatsPalmgren who is looking at fixing calc().

**Review by emilio — APPROVED** [review]

File: css/css-text/text-indent/text-indent-percentage-003.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (frivoal) [reply css/css-text/text-indent/text-indent-percentage-003.html]: I planned to file the bug after the test got approved, but @MatsPalmgren did it first

File: css/css-text/text-indent/text-indent-percentage-003.html (hunk 1 above)
Comment (frivoal) [reply css/css-text/text-indent/text-indent-percentage-003.html]: How is that: https://github.com/web-platform-tests/wpt/pull/11373/commits/d25a341c1e87e040ce092887ed89ef2156b6c189 ?

**Comment by szager-chromium** [comment]
It appears that this preexisting test checks for the old behavior:

css/css-text/text-indent/text-indent-percentage-001.xht

Are you planning to update it?

**Comment by frivoal** [comment]
handled in https://github.com/web-platform-tests/wpt/pull/13721

**Pull request merged**